youtube_dl/extractor/localnews8.py

```diff
@@ -0,0 +1,29 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+
+
+class LocalNews8IE(InfoExtractor):
+    _VALID_URL = r'http?://(?:www\.)?localnews8\.com/.+?/(?P<id>[0-9]+)'
```

Comment:
OK, I've changed it. But as of now, accessing a site on the webpage using https results in a SSL certificate error or, with `--no-check-certificates`, a 503 service unavailable. Should I add a statement that catches these errors and change the protocol to http?